### ISSUE TITLE
Incompatible with the clean task

### ISSUE THREAD

**Issue by invpos** [Digithurst/gradle-truststore-plugin#1]
Since the temporary customKeystore is forced to be in the build directory, the plugin is incompatible with the clean task. In other words, it is impossible to execute a gradle command like:

./gradlew a clean b

where "b" is the task that needs the services of this plugin.

Please either move the custom keystore out of the build directory, or allow it to be configured so that I might do so. In the mean time, I am aware of the work-around:

./gradlew a clean && ./gradlew b

Thanks!

**Comment by reitzig** [comment]
Should be very simple to make this configurable; see [here](https://github.com/Digithurst/gradle-truststore-plugin/blob/master/src/main/kotlin/com/digithurst/gradle/truststore/TruststorePlugin.kt#L31).

I don't currently have the capacity to work on this; care to provide a PR?

For a release, you'll need @msiko, anyway, as the requisite build server is at my _previous_ employer.

**Comment by invpos** [comment]
Nope, not me. Not a *.kt coder.  You don't want me poking around in your code! It was all I could do to confirm my suspicions as to how my build was failing. I will be satisfied with the work-around until the capacity [apparates](https://en.wiktionary.org/wiki/apparate#Verb).

**Comment by jiffle** [comment]
Just for anyone else who encounters this problem, using `clean` with this plugin fails the build with the following error:

```
Execution failed for task ':compileKotlin'.
 > Could not resolve all files for configuration ':compileClasspath'.
    > Could not resolve <SOME_DEPENDENCY>.
     Required by:
         project :
      > Could not resolve <SOME_DEPENDENCY>.
         > Could not get resource '<SOME_REPOSITORY>/<SOME_DEPENDENCY_PATH>'.
            > org.apache.http.ssl.SSLInitializationException: <PROJECT_PATH>/build/truststores/cacerts (No such file or directory)
```

Workaround is to run clean in one gradle command and the rest of the command in the other, as noted in OP

Wasted a lot of time trying to get to the root of the cause of this issue - surely there's an easy fix with recreating the cacerts path when it's needed rather than after evaluation?
